### DISCUSSION TITLE
Incorrect rendering of HTML info in post_metadata.html file

### DISCUSSION THREAD

**Discussion by horaciomorenom** [pmichaillat/hugo-website#1] [Q&A] [answered]
I just downloaded the template and I set it up and managed to get it to post on my browser. However, when opening any content file, such as the paper file, I get the following result:

![image](https://github.com/pmichaillat/hugo-website/assets/83434185/675e9aec-1b24-49b1-a3bd-86ef8592e9e6)

I don't see anything wrong with the post_meta.html file or the single.html default layout. Does anyone have any idea what the issue might be?

**Comment by pmichaillat** [comment, marked as answer]
I finally have an answer, thanks to @purerstamp.

The issue occurred because of changes in Hugo after version 0.120. I was using an older version of Hugo so could not replicate the problem. The issue was also described on the PaperMod repo and fixed there: https://github.com/adityatelange/hugo-PaperMod/issues/1325.

The issue was caused by the `delimit` function, and was fixed by adding `safeHTML`. See pull request #2.
